packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_place_auction_bid.test.ts

```diff
@@ -219,31 +225,42 @@ async function setup (): Promise<void> {
   await alice.generate(1)
   await tGroup.waitForSync()
 
+  const currentHeight = await bob.rpc.blockchain.getBlockCount()
+  const tslaTotalInterest = tslaInterestPerBlock.multipliedBy(currentHeight - tslaTakeLoanBlockHeight)
```

Comment:
fix height calculation

Reply:
also move `.decimalPlaces(8, BigNumber.ROUND_CEIL)` to this line. i.e keep the total interest ceiled to 8 dec places. then no need to do it again in the following line LN:230

Reply:
you are right but in this case i think i will keep the full dp in `tslaTotalInterest` because this value will be replied upon when calculating `tslaTotalInterestFrozen`
slightly more convenient to just add on addition interest to it
`const tslaTotalInterestFrozen = tslaTotalInterest.plus(tslaInterestPerBlock.multipliedBy(5))`

i will though change the loan amount calculation to 
`const tslaLoanAmountBefore = new BigNumber(tslaLoanAmount).plus(tslaTotalInterest.decimalPlaces(8, BigNumber.ROUND_CEIL))`

decimalplace before plus-ing. as it is also similar to the Ain code too.

Reply:
still needs to do -> https://github.com/DeFiCh/jellyfish/pull/947/files#r784510112
Better to use the same for consistency.